server/src/main/java/org/opensearch/cluster/coordination/JoinHelper.java

```diff
@@ -352,6 +359,10 @@ public void handleException(TransportException exp) {
                         FailedJoinAttempt attempt = new FailedJoinAttempt(destination, joinRequest, exp);
                         attempt.logNow();
                         lastFailedJoinAttempt.set(attempt);
+                        if (exp instanceof RemoteTransportException && (exp.getCause() instanceof NodeDecommissionedException)) {
+                            logger.info("local node is decommissioned. Will not be able to join the cluster");
```

Comment:
Log exp or atleast the message

Reply:
exception is logged in the method. Will add the msg